eclipse-theia/theia-blueprint
# No app icon for AppImage on Linux #296 [ jenkins ]

#### What it does

* consume image from applications/electron/resources/icons/WindowIcon/

fixes #296

Contributed on behalf of STMicroelectronics

Signed-off-by: Johannes Faltermeier <jfaltermeier@eclipsesource.com>


#### How to test
Build blueprint on linux
Check whether the application has a window image when started with `yarn electron start` and when started via the AppImage

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
